mobilecoind/api/proto/mobilecoind_api.proto

```diff
@@ -443,6 +444,24 @@ message GetBlockInfoResponse {
     uint64 txo_count = 2;
 }
 
+// Get more detailed information about a downloaded block
+message GetBlockDetailsRequest {
+    uint64 block = 1;
+}
+message GetBlockDetailsResponse {
+    // Key images in the block
+    repeated external.KeyImage key_images = 1;
+
+    // TxOuts in the block.
+    repeated external.TxOut txos = 2;
+
+    // The block signature
+    bytes signature = 3;
+
+    // The block hash
+    bytes hash = 4;
```

Comment:
Instead of the `hash`, can this include the actual `Block` from `external.Block` as the first field? That has the hash, in addition to the block id, block number, etc.

Reply:
Done!